docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe/headless.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Project guidelines for Stripe - Headless implementation
+description: Learn how to implement Stripe using ACP
+last_updated: Nov 8, 2024
+template: howto-guide-template
+related:
+   - title: Stripe
+     link: docs/pbc/all/payment-service-provider/page.version/base-shop/third-party-integrations/stripe/stripe.html
+redirect_from:
+   - /docs/pbc/all/payment-service-provider/202311.0/third-party-integrations/stripe/install-stripe.html
+   - /docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/install-stripe.html
+   - /docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/integrate-stripe.html
```

Comment:
If you copy front matter from another article, you need to remove redirects belonging to the original article